src/index.jsx

```diff
@@ -25,11 +26,27 @@ function copyGeneratedUrl(hubUrl, app) {
     });
 }
 
+function openGeneratedUrl(hubUrl, app) {
+    const query = { active: true, currentWindow: true };
+    chrome.tabs.query(query, function (tabs) {
+        const activeTab = tabs[0];
+        if (activeTab) {
+            const parts = GitUrlParse(activeTab.url);
+            const repoUrl = `${parts.protocol}://${parts.source}/${parts.full_name}`;
+            const my_url = generateRegularUrl(hubUrl, repoUrl, parts.ref, app, parts.name + '/' + parts.filepath);
```

Comment:
variables should be camelCase, so `myUrl` not `my_url`

Reply:
I also see that this is mostly duplicating the code in `copyGeneratedUrl`. I see there are two issues here:

1. this is *also* copying the URL, in addition to opening it
2. The code duplication may become a problem. Ideally, the common part of these two functions would be refactored into another one that can be called here.